app/Http/Controllers/V2MiscController.php

```diff
@@ -0,0 +1,235 @@
+                            ->with('name', 'title')
+                            ->with('value', old('title'))
+                        )
+                        ->push(component('FormTextarea')
+                            ->with('title', trans('content.forum.edit.field.body.title'))
+                            ->with('name', 'body')
+                            ->with('value', old('title'))
+                            ->with('rows', 20)
+                        )
+                        ->push(component('FormButton')
+                            ->is('hidden')
+                            ->with('title', trans('content.create.submit.title'))
+                        )
+                        ->push(component('FormButtonProcess')
+                            ->with('id', 'ForumCreateForm')
+                            ->with('title', trans('content.create.submit.title'))
+                            ->with('processingtitle', trans('content.create.submitting.title'))
+                        )
+                    )
+                )
+            )
+
+            ->with('sidebar', collect()
+                ->push(component('Block')
+                    ->is('gray')
+                    ->with('content', collect()
+                        ->push(component('Title')
+                            ->is('smaller')
+                            ->is('red')
+                            ->with('title', trans('content.edit.notes.heading'))
+                            ->with('route', route('forum.index'))
+                        )
+                        ->push(component('Body')
+                            ->with('body', trans('content.edit.notes.body'))
+                        )
+                    ))
+            )
+
+            ->with('footer', region('Footer'))
+
+            ->render();
+    }
+
+    public function store(Request $request, Content $content = null)
+    {
+        $this->validate($request, $this->rules);
+
+        return $content->id ? $this->update($request, $content) : $this->newContent($request, $request->type);
+    }
+
+    public function newContent($request, $type)
+    {
+        $content = Content::create([
+            'user_id' => Auth::user()->id,
+            'title' => $request->title,
+            'body' => $request->body,
+            'type' => $type,
+            'status' => 1,
+        ]);
+
+        Log::info('New content added', [
+            'user' =>  Auth::user()->name,
+            'title' =>  $request->title,
+            'type' =>  $type,
+            'body' =>  $request->body,
+            'link' => ($type != 'photo') ? route("$type.show", [$content]) : '',
+        ]);
+
+        return redirect()
+            ->route(''.$type.'.index')
+            ->with('info', trans('content.store.status.'.config("content_$type.store.status", 1).'.info', [
+                'title' => $content->title,
+            ]));
+    }
+
+    public function edit(Content $content)
```

Comment:
What is passed here? id or are you using https://laravel.com/docs/5.4/routing#route-model-binding ? We do not (yet) use it elsewere. I'd use passing id's like we always have done https://github.com/tripikad/trip2/blob/master/app/Http/Controllers/V2BlogController.php#L193 etc and create a new PR in the future to use route model binding everywhere